lib/bounces.js

```diff
@@ -59,7 +59,13 @@ module.exports = function (log, error) {
       }
       return P.each(recipients, function (recipient) {
         var email = recipient.emailAddress
-        log.info({ op: 'handleBounce', email: email, bounce: !!message.bounce })
+        log.info({
+          op: 'handleBounce',
+          email: email,
+          bounce: !!message.bounce,
+          diagnosticCode: recipient.diagnosticCode,
+          status: recipient.status
```

Comment:
According to docs, look like there is a `action` value in the Bounce Object? Worthwhile to add?